src/Formats/ProtobufSerializer.h

```diff
@@ -47,6 +47,17 @@ class ProtobufSerializer
         bool flatten_google_wrappers,
         ProtobufReader & reader);
 
+    static std::unique_ptr<ProtobufSerializer> create(
+        const Strings & column_names,
+        const DataTypes & data_types,
+        std::vector<size_t> & missing_column_indices,
+        const google::protobuf::Descriptor * descriptor,
+        bool with_length_delimiter,
+        bool with_envelope,
+        bool flatten_google_wrappers,
+        ProtobufReader & reader,
+        bool use_confluent);
```

Comment:
Can you explain why we need it? What is the difference when `use_confluent=true`?

Reply:
This flag indicates that we don't need to read all rows in the current iteration (we read just 1 row)